cmd/podman/image.go

```diff
@@ -20,16 +20,16 @@ var (
 var imageSubCommands = []*cobra.Command{
 	_historyCommand,
 	_imageExistsCommand,
-	_inspectCommand,
 	_imagesCommand,
+	_importCommand,
+	_inspectCommand,
 	_pruneImagesCommand,
-	_pushCommand,
```

Comment:
Did you mean to remove pushCommand?

Reply:
yes, push is not avail for remote I think